### DISCUSSION TITLE
How does DIA-NN calculate FDR?

### DISCUSSION THREAD

**Discussion by cframpas** [vdemichev/DiaNN#1225] [Q&A]
I have searched through DIA-NN 1.8 12 files first by using the MBR option and then without. With MBR, I have a higher number of precursors but less protein IDs than without MBR. Could you please explain the results? 
Thank you!

**Comment by vdemichev** [comment]
Hi,

There's a mechanism in DIA-NN which prevents FDR inflation when using MBR. This can, under certain circumstances, lead to lower numbers of proteins when doing MBR. This usually means that the library used is already quite experiment-specific.

Best,
Vadim

**Reply by cframpas** [reply]
Hi Vadim, 

Thank you for your reply, much appreciated. 

Best wishes,
Cecile
